sdk/communication/azure-communication-callautomation/azure/communication/callautomation/_shared/models.py

```diff
@@ -23,6 +23,7 @@ class CommunicationIdentifierKind(with_metaclass(CaseInsensitiveEnumMeta, str, E
     COMMUNICATION_USER = "communication_user"
     PHONE_NUMBER = "phone_number"
     MICROSOFT_TEAMS_USER = "microsoft_teams_user"
+    MICROSOFT_BOT = "microsoft_bot"
```

Comment:
Can we update the documentation of the `"unknown"` kind to let people know that they shouldn't rely on the unknown classification, as it become "known" in later SDK releases. Instead that should only rely on the presence of the raw_id.

Reply:
I updated the documentation. Please tell me if you like the wording :)

Reply:
I left a comment on the updated docstring below - though we should also add something similar to the Enum docstring as well :)